plugins/catalog/src/components/SystemDiagramCard/SystemDiagramCard.test.tsx

```diff
@@ -48,8 +48,8 @@ describe('<SystemDiagramCard />', () => {
       apiVersion: 'v1',
       kind: 'System',
       metadata: {
-        name: 'my-system2',
```

Comment:
Removed these prefixes as a workaround, leaving these in would exceed the max character count, truncate and then the test would break. Updating these prefixes seemed like the least intrusive fix, rather than update assertions. I also considered having a default value within the function that applies the truncation, however it would need to be proxied through from the top level component and muddied some of the function signatures.